packages/extension-polkagate/src/popup/notification/NotificationItem.tsx

```
@@ -0,0 +1,193 @@
+  const sanitizeFloatAmount = useCallback((amount: string) => {
+    const dotIndex = amount.indexOf('.');
+
+    return dotIndex === -1
+      ? amount
+      : amount.slice(0, dotIndex + 3);
+  }, []);
+
+  const sanitizeBNAmount = useCallback((amount: string, decimal: number) => {
+    const bnAmount = Number(amount);
+
+    return isNaN(bnAmount)
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)